example/matchTag/input.go

```diff
@@ -0,0 +1,14 @@
+package example
```

Comment:
The directory of this file should be the same format as the other directories. match-tag